packages/cli/src/commands/up.ts

```diff
@@ -51,14 +52,26 @@ export default class Up extends MachineCreationDriverCommand<typeof Up> {
       insecureSkipVerify: flags['insecure-skip-verify'],
     }
 
-    const { hostKey, clientId, rootUrl } = await tunnelServerHello({
-      tunnelingKey,
-      knownServerPublicKeys: pStore.knownServerPublicKeys,
-      tunnelOpts,
-      log: this.logger,
+    let clientId = ''
+    let rootUrl = ''
+    let hostKey: Buffer = Buffer.from('')
+
+    await withSpinner(async spinner => {
+      spinner.start('Initializing tunnel connection...')
```

Comment:
there's no need to call `spinner.start`, `withSpinner` accepts a `text` option (part of the Ora options)